src/pages/SetPasswordPage.js

```diff
@@ -99,26 +100,26 @@ class SetPasswordPage extends Component {
                     shouldShowWelcomeText
                     welcomeText={this.props.translate('setPasswordPage.passwordFormTitle')}
                 >
-                    <View style={[styles.mb4]}>
-                        {/* The prop onSubmitEditing is required, but it needs to stay as a no-op because the form is submitted and validated from the button below */}
-                        <NewPasswordForm
-                            onSubmitEditing={() => {}}
```

Comment:
Why did you remove this prop?

Reply:
TextInput has onSubmitEditing, this props was used to submit the form when Enter is pressed. As the implementation now has changed, and we are handling it with a custom keyboard listener for enter key inside FormSubmit component. Therefore, that is not required.